README.md

```diff
@@ -2,7 +2,7 @@
 
 [![CircleCI](https://dl.circleci.com/status-badge/img/gh/secretflow/yacl/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/secretflow/yacl/tree/main)
 
-YACL (Yet Another Crypto library) is a C++ library that contains modules and utilities which other SecretFlow code depends on.
+YACL (Yet Another Crypto library) is a C++ library that contains modules and utilities which other SecretFlow codes depend on.
```

Comment:
I don't think "codes" is correct here.

e.g. we usually say "other peoples' code" instead of "other peoples' codes".